ui/pages/confirm-deploy-contract/confirm-deploy-contract.component.js

```diff
@@ -17,26 +26,56 @@ export default class ConfirmDeployContract extends Component {
     const { txData: { origin, txParams: { data } = {} } = {} } = this.props;
 
     return (
-      <div className="confirm-page-container-content__data">
-        <div className="confirm-page-container-content__data-box">
-          <div className="confirm-page-container-content__data-field">
-            <div className="confirm-page-container-content__data-field-label">
+      <Box color={Color.textAlternative} padding={4}>
+        <Box
+          backgroundColor={Color.backgroundAlternative}
+          padding={4}
+          overflowWrap={OVERFLOW_WRAP.BREAK_WORD}
```

Comment:
`overflowWrap` prop doesn't exists on the Box component did you meant to use a `Text` component? Or you may need to add the css here for overflow:hidden

```suggestion
```

Reply:
Fixed 👍